lib/kino_maplibre/map_cell.ex

```diff
@@ -511,4 +516,12 @@ defmodule KinoMapLibre.MapCell do
     valid_lat? = lat >= -90 and lat <= 90
     if valid_lng? and valid_lat?, do: {lng, lat}
   end
+
+  defp normalize_geocode_id(query) do
+    query
+    |> String.downcase()
+    |> String.normalize(:nfd)
+    |> String.replace(~r/[^A-z\s]/u, "")
```

Comment:
```suggestion
    |> String.replace(~r/[^a-zA-Z\s]/u, "")
```

Between `A` and `z` there are characters like `[`, `]`, `^` and I assume we don't want them in the range?

Reply:
Yep! Thanks ❤️ and I completely forgot the postal codes 🙈